pkg/cli/environment.go

```diff
@@ -91,5 +91,8 @@ func (s *EnvSettings) EnvVars() map[string]string {
 		"HELM_REGISTRY_CONFIG":   s.RegistryConfig,
 		"HELM_REPOSITORY_CACHE":  s.RepositoryCache,
 		"HELM_REPOSITORY_CONFIG": s.RepositoryConfig,
+		"HELM_NAMESPACE":         s.Namespace,
+		"HELM_KUBECONTEXT":       s.KubeContext,
+		"HELM_KUBECONFIG":        s.KubeConfig,
```

Comment:
How about exposing this as `KUBECONFIG` for plugins? That way invocations of`helm` (or `kubectl`) from a plug-in would automatically pick in the correct config. This would have to be conditional so that if `—kubeconfig` is not set we don‘t clobber a value set in the `KUBECONFIG`  env var

Reply:
@databus23 Sounds good! Addressed in the last commit.